app/mailers/activity_mailer.rb

```diff
@@ -251,7 +251,7 @@ def watching_notification(args = {})
       kind: Notification.kinds[:watched]
     )
     @action = @watchable.instance_of?(Question) ? '回答' : 'コメント'
-    subject = "[FBC] #{@watchable.user.login_name}さんの【 #{@watchable.notification_title} 】に#{@sender.login_name}さんが#{@action}しました。"
+    subject = "[FBC] #{@watchable.user.login_name}さんの#{@watchable.notification_title}で#{@sender.login_name}さんが#{@action}しました。"
```

Comment:
細かいですが個人的には『に』の方が自然な気がしました🙇‍♂️
```suggestion
    subject = "[FBC] #{@watchable.user.login_name}さんの#{@watchable.notification_title}に#{@sender.login_name}さんが#{@action}しました。"
```

上記を採用していただける場合、メンション側も修正が必要になると思いますので、お手数ですがご対応ください🙇‍♂️